app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -1,32 +1,30 @@
 package com.example.myapplication;
 
 import android.os.Bundle;
-
 import com.google.android.material.snackbar.Snackbar;
-
 import androidx.appcompat.app.AppCompatActivity;
-
 import androidx.navigation.NavController;
 import androidx.navigation.Navigation;
 import androidx.navigation.ui.AppBarConfiguration;
 import androidx.navigation.ui.NavigationUI;
-
 import com.example.myapplication.databinding.ActivityMainBinding;
-
 import android.view.Menu;
 import android.view.MenuItem;
-
-import java.util.Objects;
+import android.view.View;
 
 public class MainActivity extends AppCompatActivity {
     // In-app variables
     private AppBarConfiguration appBarConfiguration;
+    private ActivityMainBinding binding;
+
+    public static String teamNumber = "0";
+    public static String matchNumber = "0"; // Add this line for matchNumber
```

Comment:
Is this used anywhere? If not, please remove it.

Reply:
this is.

Reply:
never mind